README.md

```diff
@@ -56,10 +69,20 @@ create-version-file metadata.yml --outfile file_version_info.txt
 ```
 where metadata.yml is the YAML configuration file from above.
 
+To run metadata extraction from distribution call:
+```cmd
+create-version-file PackageName --source-format dist --outfile file_version_info.txt
+```
 
 #### Extracting Version Information
-As an alternative to specifying the version directly in the YAML file, there are two alternatives which may be more
-suitable, depending on the use case:
+In addition to otherwise constant project data, the version number is an
+exception that requires additional effort. Of course, it is most convenient if
+the version is managed within the distribution and can be taken
+from its metadata automatically.
```

Comment:
This section seems to suggest that having the project installed to site-packages is the preferred approach. I do not fully agree with that, as you also stated that this is probably a niche use case.

Another concern in this case is, that for the `Company` field no proper source in the distribution metadata exists, which I personally think can be a very important information for executables. 

Thus I would prefer something closer to this:

*As an alternative to specifying the version directly in the YAML file or the distribution metadata, there are two options which may be more suitable, depending on the use case.*

Reply:
please check the update. I suggested a short additional section because development within installed packages is not a niche case. This is why there is a good reason for the editable installation.

Reply:
Fine with me with the latest edit. 👍

Reply:
thanks for your patience :-)